recipes/libpq/all/conanfile.py

```diff
@@ -134,7 +139,7 @@ def _patch_sources(self):
                     "MTd": "MultiThreadedDebug",
                     "MD": "MultiThreadedDLL",
                     "MDd": "MultiThreadedDebugDLL",
-                }.get(str(self.settings.compiler.runtime))
+                }.get(msvc_runtime_flag(self))
```

Comment:
not like this

Reply:
my review suggested to remove branching

Reply:
I don't understand your comment. This is exactly what you suggested earlier